nixos/lib/test-driver/test-driver.py

```diff
@@ -544,30 +552,30 @@ def get_screen_text(self):
 
                 return ret.stdout.decode("utf-8")
 
-    def wait_for_text(self, regex):
-        def screen_matches(last):
+    def wait_for_text(self, regex: str) -> None:
+        def screen_matches(last: bool) -> bool:
             text = self.get_screen_text()
             m = re.search(regex, text)
```

Comment:
maybe bring the correct type forward, so `m = bool(re.search(regex, text))` instead of the `return m is not None`